src/main/java/com/eleks/academy/whoami/core/impl/PersistentGame.java

```diff
@@ -77,6 +73,16 @@ public SynchronousPlayer enrollToGame(String player) {
 		}
 	}
 
+	@Override
+	public void setCharacters(String player, CharacterSuggestion characters) {
+		var currentPlayer = currentState.peek().findPlayer(player).orElseThrow();
+		currentPlayer.suggestCharacter(characters);
```

Comment:
right now the state of `SuggestingCharacters` looks empty it doesn't do any useful work, but `PersistanceGame` is overloaded, much better would be if we move the logic of a suggesting into `SuggestingCharacters`, so here this method of `PersistanceGame` would look more simple like this:
```
@Override
	public void setCharacters(String player, CharacterSuggestion characters) {
		SuggestingCharacters suggesting = (SuggestingCharacters) currentState.peek();
		suggesting.setCharacters(player, characters); // TODO: here to implement find player and setiing characters
		if (suggesting.finished()) {
			currentState.add(suggesting.next());
			currentState.remove();
		}
```